src/test/ui/rfc-2008-non-exhaustive/variants_fictive_visibility.rs

```diff
@@ -0,0 +1,9 @@
+// compile-pass
```

Comment:
A longer comment explaining the purpose of this test would be neat; it feels like a head scratcher otherwise...